cli/dcoscli/package/main.py

```diff
@@ -454,6 +454,10 @@ def _search(json_, query):
                            results,
                            tables.package_search_table,
                            json_)
+    if not any(map(
+            lambda element: len(element.get('packages', [])) > 0, results
+    )):
+        sys.stderr.write('No packages found for query {}\n'.format(query))
```

Comment:
Our canonical way of writing to stderr is to raise a DCOSException.  The exception message is printed to stderr